tests/test_integration_testing.py

```diff
@@ -43,6 +43,10 @@ def test_integration_test(self):
     @patch('egcg_core.integration_testing.get_cfg')
     @patch('egcg_core.integration_testing.check_output')
     def test_reporting_app_integration_test(self, mocked_check_output, mocked_get_cfg, mocked_sleep, mocked_get):
+        # Change directory to local test folder
+        test_dir = os.path.abspath(os.path.dirname(__file__))
+        os.chdir(test_dir)
```

Comment:
I'm not generally a fan of changing working dir during tests because it can have side effects on other tests depending on whether some/all are being run, but I don't really have a problem here with making sure we're in `tests/`, because that's usually where we run them anyway.

Reply:
I've made sure the test reverts back to wherever we were before.
I does not have to be in the test folder for the test to work but because we creating a directory that might not get cleaned up if the test fails I'd rather we are so we don't pollute our environment.